src/assets/javascript/components/buttom.js

```diff
@@ -0,0 +1,6 @@
+function buttom(textContent) {
```

Comment:
Hey @Willian1661 other thing, you should move 

components to `src/components` 👀

Reply:
i moved now, but why? is default?

Reply:
https://dev.to/vadorequest/a-2021-guide-about-structuring-your-next-js-project-in-a-flexible-and-efficient-way-472

Reply:
ah true, i forgot :0